src/sql/workbench/contrib/connection/browser/connection.contribution.ts

```diff
@@ -123,6 +123,18 @@ configurationRegistry.registerConfiguration({
 			'default': 25,
 			'description': localize('sql.maxRecentConnectionsDescription', "The maximum number of recently used connections to store in the connection list.")
 		},
+		'sql.defaultAuthenticationType': {
```

Comment:
This is the new configuration setting. Users can edit and set this enum. via the ADO UI.